src/wallet/rpcwallet.cpp

```diff
@@ -3333,6 +3371,10 @@ UniValue z_getaddressforaccount(const UniValue& params, bool fHelp)
                     strErr = tfm::format(
                         "Error: one or more of the requested receiver types does not have a corresponding spending key in this account.");
                     break;
+                case UnifiedAddressGenerationError::ReceiverTypeNotSupported:
+                    strErr = tfm::format(
+                        "Error: P2SH addresses can not be created by zcashd.");
```

Comment:
```suggestion
                        "Error: P2SH addresses can not be created using this RPC method.");
```

Reply:
Are there actually ways to create P2SH addresses in zcashd?

Reply:
Yes, using the `addmultisigaddress` RPC API method.